tests/agent-scenarios.test.ts

```
@@ -0,0 +1,351 @@
+    console.log(validation.passed ? '✅ PASSED' : '❌ FAILED');
+    if (!validation.passed) {
+      console.log(`   Error: ${validation.message}`);
+    }
+    
+    return validation;
+  } finally {
+    await cleanupDatabase();
+  }
+}
+
+// Batch test runner for all scenarios
```

Biome (1.9.4)

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)